nixie/cli.py

```
@@ -117,6 +119,26 @@ async def wait_for_proc(
     return success, stderr
 
 
+async def _run_mermaid_cli(
+    cmd: list[str],
+    sem: asyncio.Semaphore,
+    path: Path,
+    idx: int,
+    timeout: float,
```

Ruff (0.12.2)

Async function definition with a `timeout` parameter

(ASYNC109)